plugins/modules/route53.py

```diff
@@ -604,6 +629,33 @@ def main():
         'HealthCheckId': health_check_in,
         'SetIdentifier': identifier_in,
     })
+
+    if geo_location:
+        continent_code = geo_location.get('continent_code')
+        country_code = geo_location.get('country_code')
+        subdivision_code = geo_location.get('subdivision_code')
+
+        if geo_location and not module.params.get('identifier'):
+            module.fail_json(changed=False, msg='To use geo_location please specify identifier.')
```

Comment:
can this be simplified by making `geo_location` and `identifier` `required_together` ?

Reply:
Fixed.